### DISCUSSION TITLE
Graph Scale

### DISCUSSION THREAD

**Discussion by JohnHawkes** [jneilliii/OctoPrint-BedLevelVisualizer#437] [General]
Is it possible to change the scale of the plot. At the moment the -2 to 2 mm range is a bit big for really fine tuning my bed.

Many thanks

John

**Comment by jneilliii** [comment]
There is an option in the settings, z Limits....if you set it to auto,auto it will scale based on the points provided by the firmware, but may look odd when rendered.
